app/Http/Controllers/Api/AcarsController.php

```diff
@@ -71,6 +72,16 @@ function ($pirep) {
         return PirepResource::collection($pireps);
     }
 
+    /**
+     * Gets all the supported acars sources in the AcarsSource Enum
+     *
+     * @return void
+     */
+    public function get_supported_acars_sources()
+    {
+        return response()->json(AcarsSource::$labels);
+    }
+
```

Comment:
I don't think that we would need this at all, as the sources will be known and static (enums), no need to have a json response to get the sources via API.

The module developer will know the proper enum to use anyway. (documentation)